2018-06-11 10:18:58,559 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 10:18:58,727 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 10:21:19,188 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 10:21:19,352 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 10:24:51,414 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 10:24:51,514 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 10:24:55,322 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 10:24:55,434 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 10:27:50,423 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 10:27:50,591 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 10:28:20,142 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 10:28:20,237 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 10:29:51,126 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 10:29:51,233 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 10:30:39,072 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 10:30:39,188 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 11:29:42,686 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:29:42,741 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 2878
2018-06-11 11:32:07,895 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:32:09,497 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:32:10,081 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:32:10,273 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:32:10,777 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:43:08,677 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:45:00,829 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:45:02,793 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:46:29,470 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:58:42,547 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:58:44,051 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:58:44,203 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:58:44,363 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:58:44,507 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:58:44,650 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 11:59:21,271 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:00:55,551 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:05:34,838 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:08:48,532 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 500 75063
2018-06-11 13:12:02,819 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 500 72480
2018-06-11 13:12:27,666 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:13:21,002 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:13:40,111 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 500 72525
2018-06-11 13:13:47,347 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 500 72525
2018-06-11 13:14:31,952 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 500 72541
2018-06-11 13:14:58,661 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:15:07,796 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:16:28,753 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:16:45,652 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:27:44,186 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:28:08,723 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:29:16,093 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:29:44,412 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:29:58,789 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:37:28,735 ERROR    "GET /get_user_list/ HTTP/1.1" 500 72822
2018-06-11 13:38:05,755 ERROR    "GET /get_user_list/ HTTP/1.1" 500 72446
2018-06-11 13:38:54,062 ERROR    "GET /get_user_list/ HTTP/1.1" 500 72648
2018-06-11 13:39:25,332 ERROR    "GET /get_user_list/ HTTP/1.1" 500 72670
2018-06-11 13:39:31,962 INFO     "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:41:38,316 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:41:39,118 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:41:39,664 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:42:03,500 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:42:08,359 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:42:08,863 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:42:09,213 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:42:09,366 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:43:31,024 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:43:31,511 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:43:31,854 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:44:07,210 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:45:49,582 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:46:13,074 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:47:19,090 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 500 69216
2018-06-11 13:47:26,848 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 500 72829
2018-06-11 13:48:10,621 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:48:33,053 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 13:49:02,197 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 15:10:36,926 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 15:33:03,405 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 16:38:47,588 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_user_list/ HTTP/1.1" 200 78
2018-06-11 16:38:56,562 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET / HTTP/1.1" 404 2827
2018-06-11 16:39:07,392 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_index_setting/ HTTP/1.1" 200 97
2018-06-11 17:12:57,792 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /get_index_setting/ HTTP/1.1" 200 97
2018-06-12 16:30:10,666 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 500 69247
2018-06-12 16:31:49,882 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 500 69240
2018-06-12 16:32:35,307 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET / HTTP/1.1" 404 2934
2018-06-12 16:32:38,632 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 500 69335
2018-06-12 16:33:03,611 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 500 81407
2018-06-12 16:35:52,004 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 500 77418
2018-06-12 16:36:57,089 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 500 77418
2018-06-12 16:37:21,431 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 500 77418
2018-06-12 16:38:37,964 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 200 60
2018-06-12 16:38:52,783 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /upload/ HTTP/1.1" 200 60
2018-06-12 16:47:15,983 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 500 60689
2018-06-12 16:47:18,395 ERROR    C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 500 60289
2018-06-12 16:47:31,622 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 200 3
2018-06-12 16:47:57,130 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 200 2
2018-06-12 16:48:04,396 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 200 2
2018-06-12 17:54:17,516 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 200 2
2018-06-12 17:56:37,991 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 200 136
2018-06-12 17:57:46,580 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 200 16
2018-06-12 17:57:55,932 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin HTTP/1.1" 301 0
2018-06-12 17:57:55,961 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/ HTTP/1.1" 302 0
2018-06-12 17:57:56,066 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1857
2018-06-12 17:57:56,190 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/css/login.css HTTP/1.1" 200 1203
2018-06-12 17:57:56,191 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/css/responsive.css HTTP/1.1" 200 17894
2018-06-12 17:57:56,222 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
2018-06-12 17:57:56,230 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
2018-06-12 17:57:56,239 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
2018-06-12 17:57:56,303 WARNING  C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /favicon.ico HTTP/1.1" 404 3095
2018-06-12 17:58:01,625 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
2018-06-12 17:58:01,661 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/ HTTP/1.1" 200 8143
2018-06-12 17:58:01,678 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
2018-06-12 17:58:01,688 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
2018-06-12 17:58:01,689 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
2018-06-12 17:58:01,690 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
2018-06-12 17:58:04,549 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/blog/site/ HTTP/1.1" 200 4359
2018-06-12 17:58:04,579 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
2018-06-12 17:58:04,579 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6897
2018-06-12 17:58:04,579 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/core.js HTTP/1.1" 200 7134
2018-06-12 17:58:04,579 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
2018-06-12 17:58:04,583 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/jsi18n/ HTTP/1.1" 200 7684
2018-06-12 17:58:04,588 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 258648
2018-06-12 17:58:04,589 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/actions.js HTTP/1.1" 200 6502
2018-06-12 17:58:04,589 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1538
2018-06-12 17:58:04,597 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/urlify.js HTTP/1.1" 200 8729
2018-06-12 17:58:04,599 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
2018-06-12 17:58:04,636 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
2018-06-12 17:58:05,844 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/blog/site/1/change/ HTTP/1.1" 200 5294
2018-06-12 17:58:05,871 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
2018-06-12 17:58:05,873 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/js/change_form.js HTTP/1.1" 200 712
2018-06-12 17:58:05,874 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/css/forms.css HTTP/1.1" 200 8173
2018-06-12 17:58:05,874 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/jsi18n/ HTTP/1.1" 200 7684
2018-06-12 17:58:05,881 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
2018-06-12 17:58:18,157 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "POST /set_index/ HTTP/1.1" 200 16
2018-06-12 17:58:20,791 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/blog/site/1/change/ HTTP/1.1" 200 5294
2018-06-12 17:58:20,806 INFO     C:\Users\Administrator\AppData\Local\Programs\Python\Python35\lib\site-packages\django\core\servers\basehttp.py[line:124] "GET /admin/jsi18n/ HTTP/1.1" 200 7684
